api/services/reporting/report_generator.py

```
@@ -0,0 +1,207 @@
+from weasyprint import HTML, CSS
+
+from api.schemas.models import ResultadoVerificacao, ResultadoVerificacaoPDF
+from api.schemas.audit_models import PDFAuditReport
+
+logger = logging.getLogger(__name__)
+
+
+class ReportGenerator:
+    """Service for generating PDF reports from verification results"""
+
+    def __init__(self):
```

Ruff (0.14.5)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)